DESCRIPTION

```diff
@@ -73,4 +73,6 @@ Encoding: UTF-8
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.1.1
 SystemRequirements: pandoc
-Remotes: r-lib/downlit
+Remotes:  
```

Comment:
Given that this just an example, and bslib will probably be released before pkgdown, I wouldn't bother bumping the required version here.